src/bilder/images/edxapp/files/edx_web_playbook.yml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Should this be called `edx_app_playbook.yml`?

Reply:
It's being matched based on the `web` or `worker` var inputs in the packer config.